src/file-1.xoo

```diff
@@ -0,0 +1,16 @@
+// dummy line 1
+// dummy line 2
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// dummy line 7
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// dummy line 11
+// dummy line 12
+// dummy line 13
+// dummy line 14
+// dummy line 15
+// dummy line 16
+// violate HasTag rule: xoo on line 17
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:39181/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:39181/project/issues?id=sample&pullRequest=6066&issues=AWeiXA9mbZxADX66CDTs&open=AWeiXA9mbZxADX66CDTs)

[ISSUE_KEY]: AWeiXA9mbZxADX66CDTs